app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -29,6 +29,7 @@ function show_interface_modal(modal_content) {
   modal_window.modal({'show': true});
 
   modal_window.find('a[rel="popover-modal"]').popover({html: true});
+  $('fieldset.compute_attributes select').select2();
```

Comment:
bootstrap-css already added in https://github.com/theforeman/foreman/pull/2415/files#diff-f859e2848b07324f808ce1a5ccd9fcbaR7